### PR TITLE
Fix flaky go test TestGetMoveQueueItemsComboSubmittedDate

### PR DESCRIPTION
**Pull request by donaldthai**
## Description

The go test, TestGetMoveQueueItemsComboSubmittedDate(), was showing flakiness with some builds. This fixes the test by separating the test into two tests.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/166616637) for this change

### CODE REVIEW

File: pkg/handlers/internalapi/move_queue_items_test.go [hunk 1]
```diff
@@ -168,25 +168,69 @@ func (suite *HandlerSuite) TestGetMoveQueueItemsComboSubmittedDate() {
...
+
+	// Then: Expect a 200 status code
+	okResponse := showResponse.(*queueop.ShowQueueOK)
+
+	suite.Equal(1, len(okResponse.Payload))
+
+	moveQueueItem := okResponse.Payload[0]
+	resultPPMDate := *handlers.FmtDateTimePtrToPopPtr(moveQueueItem.SubmittedDate)
+
+	suite.Equal(ppmSubmitDate.Format(time.UnixDate), resultPPMDate.Format(time.UnixDate))
```
Comment (ntwyman) [inline pkg/handlers/internalapi/move_queue_items_test.go]: Why is it necessary to compare these as strings, rather than comparing the underlying values?

**Comment by donaldthai** [comment]
> This looks good, but it's a little hard to parse why the test was failing before. Could you shed some light on that @donaldthai?

Yea, here's the slack thread with more info: https://defensedigitalservice.slack.com/archives/G8P7UD05U/p1560197886165700

File: pkg/handlers/internalapi/move_queue_items_test.go (hunk 1 above)
Comment (donaldthai) [reply pkg/handlers/internalapi/move_queue_items_test.go]: I had some trouble comparing just the values directly because of the `Time` monotonic value and location (timezone) value since we convert the swagger time type to pop time type.

https://golang.org/pkg/time/#hdr-Monotonic_Clocks
https://golang.org/pkg/time/#Location

Here's a failed example:
```golang
--- FAIL: TestHandlerSuite (0.61s)
    --- FAIL: TestHandlerSuite/TestGetMoveQueueItemsComboSubmittedDateHHG (0.61s)
        move_queue_items_test.go:257: 
            	Error Trace:	move_queue_items_test.go:257
            	Error:      	Not equal: 
            	            	expected: time.Time{wall:0xbf38697347fc3468, ext:604461951, loc:(*time.Location)(0x3d3fb00)}
            	            	actual  : time.Time{wall:0x7fc3468, ext:63695955277, loc:(*time.Location)(nil)}
            	            	
            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -1,2 +1,2 @@
            	            	-(time.Time) 2019-06-12 16:54:37.133969 +0000 UTC m=+0.604461951
            	            	+(time.Time) 2019-06-12 16:54:37.133969 +0000 UTC
            	            	 
            	Test:       	TestHandlerSuite/TestGetMoveQueueItemsComboSubmittedDateHHG
```

Converting to string gave me the values I wanted. Eg: Convert to string unix time ""Wed Jun 12 17:13:31 UTC 2019"

I think if we want to include the time location information, we should probably revisit how we're parsing the swagger time to pop time.

Comment (donaldthai) [reply pkg/handlers/internalapi/move_queue_items_test.go]: I'll put in a chore to investigate @ntwyman
